azure-pipelines/pull-request-validation/pr-msal.yml

```diff
@@ -30,7 +30,7 @@ resources:
   - repository: common
     type: github
     name: AzureAD/microsoft-authentication-library-common-for-android
-    ref: dev
+    ref: robert/empty-correlation-id-fixes
```

Comment:
Revert it back before the merge.

Reply:
Why do we have this here?

Reply:
I cannot change the targe branch in the DevOps portal thus change the command here.